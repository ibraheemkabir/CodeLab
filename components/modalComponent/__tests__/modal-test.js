import React from 'react';
import {shallow} from 'enzyme'
import renderer from 'react-test-renderer';
import Modal from '../modal';

it(`renders correctly`, () => {
  const modal = renderer.create(<Modal/>);

  expect(modal).toMatchSnapshot();
});

it(`renders correctly`, () => {
  const props = {
    setModalVisible: jest.fn()
  }
  const modal = shallow(<Modal {...props} />)
  const simulate = modal.find('TouchableHighlight').simulate('press')
});
