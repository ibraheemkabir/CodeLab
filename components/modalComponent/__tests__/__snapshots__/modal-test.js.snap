// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  transparent={true}
  visible={true}
>
  <View
    style={
      Object {
        "alignContent": "center",
        "alignItems": "center",
        "flex": 1,
        "height": 30,
        "justifyContent": "center",
        "marginTop": 22,
        "shadowColor": "black",
        "shadowOffset": Object {
          "height": -4,
        },
      }
    }
  >
    <View
      style={
        Object {
          "alignContent": "center",
          "alignItems": "center",
          "backgroundColor": "white",
          "flexDirection": "column",
          "height": 170,
          "justifyContent": "center",
          "shadowColor": "black",
          "shadowOffset": Object {
            "height": -4,
          },
          "shadowOpacity": 0.5,
          "width": 400,
        }
      }
    >
      <View
        style={
          Object {
            "alignContent": "flex-start",
            "alignItems": "flex-start",
            "justifyContent": "flex-start",
            "paddingRight": 15,
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={Object {}}
        >
          <Text />
        </View>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "height": 100,
              "width": 100,
            }
          }
        />
        <Image
          source={1}
          style={
            Object {
              "height": 100,
              "width": 100,
            }
          }
        />
        <Image
          source={1}
          style={
            Object {
              "height": 100,
              "width": 100,
            }
          }
        />
      </View>
    </View>
  </View>
</Modal>
`;
